### ISSUE TITLE
Enum description doesn't support markdown

### ISSUE THREAD

**Issue by reech-florian** [magidoc-org/magidoc#207]
### Describe the bug

The bug is similar to #186 . On GraphQL enums values, Magidoc doesn't handle markdown formatting.

![image](https://user-images.githubusercontent.com/71769614/232540089-fb73e8d8-517f-4ca6-8f97-beaeae1f9dd7.png)

### Reproduction

schema.graphql

```graphql
"The **dummy** enum"
enum DummyEnum {
  "A **value** of the enum."
  DummyTest

  """
  Another **value** of __the enum__.
  """
  AnotherDummyTest
}
```

### System Info

```shell
System:
    OS: Linux 5.19 Ubuntu 22.04.2 LTS 22.04.2 LTS (Jammy Jellyfish)
    CPU: (12) x64 Intel(R) Core(TM) i7-9850H CPU @ 2.60GHz
    Memory: 18.27 GB / 31.14 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
Browsers:
    Chrome: 112.0.5615.49
```

### Severity

Annoyance

**Comment by pelletier197** [comment]
You're right, there seem to be missing a markdown component [here](https://github.com/magidoc-org/magidoc/blob/main/packages/starters/carbon-multi-page/src/lib/components/type/EnumType.svelte). I'll add it when I get some time this week.

If this is urgent, contributions are always appreciated. 🙂

**Comment by reech-florian** [comment]
Hello, thank you for your quick response. I will try to do a PR if I can !

**Comment by pelletier197** [comment]
This was added in 3.6.4. Thanks for raising the issue!

**Comment by reech-florian** [comment]
Oh, I didn't have time to correct it on my side, thanks for your reactivity!
